README.md

```diff
@@ -1,22 +1,31 @@
-# PostHTML Content
+[![NPM][npm]][npm-url]
+[![Dependencies][deps]][deps-url]
+[![DevDependencies][devdeps]][devdeps-url]
```

Comment:
The test && coverage badges are in the table under the headline in the README. rm the table and it to them top again?